frontend/fingertips-frontend/components/organisms/LineChartTable/index.tsx

```diff
@@ -1,46 +1,170 @@
+  paddingLeft: '0',
+});
+
+const StyledGreyHeader = styled(StyledAlignRightHeader)({
+  backgroundColor: LIGHT_GREY,
+});
+
+const StyledTable = styled(Table)({
+  marginBottom: '0',
+});
+
+const StyledTableCell = styled(Table.Cell)(typography.font({ size: 16 }), {
+  paddingRight: '0',
+});
+
+const StyledAlignLeftTableCell = styled(StyledTableCell)({
+  textAlign: 'left',
+});
+
+const StyledAlignRightTableCell = styled(StyledTableCell)({
+  textAlign: 'right',
+});
+
+const StyledBenchmarkValueTableCell = styled(StyledTableCell)({
+  backgroundColor: LIGHT_GREY,
+  borderTop: 'solid white 0.09375em',
+});
+
+const mapToTableData = (areaData: HealthDataForArea): LineChartTableData[] =>
+  areaData.healthData.map((healthPoint) => ({
+    period: healthPoint.year,
+    count: healthPoint.count,
+    value: healthPoint.value,
+    lower: healthPoint.lowerCi,
+    upper: healthPoint.upperCi,
+  }));
+
+function getCellHeader(heading: string): ReactNode {
+  return heading === LineChartTableHeadingEnum.BenchmarkValue ? (
+    <StyledGreyHeader key={`header-${heading}`}>{heading}</StyledGreyHeader>
+  ) : (
+    <StyledAlignRightHeader key={`header-${heading}`}>
+      {heading}
+    </StyledAlignRightHeader>
+  );
+}
+
 export function LineChartTable({ data, headings }: Readonly<TableProps>) {
-  data = sortHealthDataByDate(data);
+  const tableData = mapToTableData(data);
+  const dataSortedByPeriodDesc = tableData.toSorted(
+    (a, b) => b.period - a.period
+  );
   return (
     <div data-testid="lineChartTable-component">
-      <Table
+      <StyledTable
+        head={
+          <Table.Row>
+            <StyledAlignCenterHeader setWidth="one-quarter"></StyledAlignCenterHeader>
+            <StyledAlignCenterHeader setWidth="one-quarter"></StyledAlignCenterHeader>
+            <StyledAlignCenterHeader>{data.areaCode}</StyledAlignCenterHeader>
+            {[...Array(7).keys()].map((i) => (
+              <Table.CellHeader key={i}></Table.CellHeader>
+            ))}
+            <StyledGreyHeader>England</StyledGreyHeader>
+          </Table.Row>
+        }
+      ></StyledTable>
+      <StyledTable
         head={
           <Table.Row>
-            {headings?.map((heading) => (
-              <Table.CellHeader key={`header-${heading}`}>
-                {heading}
-              </Table.CellHeader>
+            <StyledAlignRightHeader setWidth="one-quarter"></StyledAlignRightHeader>
+            <StyledAlignRightHeader setWidth="one-quarter"></StyledAlignRightHeader>
+            {[...Array(6).keys()].map((i) => (
+              <Table.CellHeader key={i}></Table.CellHeader>
             ))}
+            <StyledConfidenceLimitsHeader>
+              95% confidence limits
+            </StyledConfidenceLimitsHeader>
+          </Table.Row>
+        }
+      ></StyledTable>
+      <StyledTable
+        head={
+          <Table.Row>
+            {headings?.map((heading) =>
+              heading === 'Period' || heading === 'Trend' ? (
```

Comment:
If you use the enum to iterate. We can use the enum here instead of hardcoding these